homeassistant/components/shelly/climate.py

```diff
@@ -126,7 +128,10 @@ def __init__(
         self.last_state: State | None = None
         self.last_state_attributes: Mapping[str, Any]
         self._preset_modes: list[str] = []
-        self._last_target_temp = 20.0
+        if coordinator.hass.config.units is US_CUSTOMARY_SYSTEM:
+            self._last_target_temp = 68.0
```

Comment:
Totally discretionary: would it make sense to turn these both into constants? Or do we anticipate them never needing to change?

Reply:
Yeah I agree, those values look bad, I'll change this.